sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/patch/PatchUtil.java

```diff
@@ -13,16 +14,12 @@
 
 public final class PatchUtil {
 
-    public static <T> JsonSerializable serializableBatchPatchOperation(T item) {
-        if (item instanceof CosmosPatchOperations) {
-            return cosmosPatchToJsonSerializable((CosmosPatchOperations) item);
-        } else {
-            throw new UnsupportedOperationException("Unknown Patch operations.");
-        }
+    public static <T> JsonSerializable serializableBatchPatchOperation(CosmosPatchOperations patchOperations, RequestOptions requestOptions) {
```

Comment:
No need of T now? Or you can keep the old implementation where ItemBatch/Bulk operations didnot had to worry about typecasting.

Reply:
This way a readable exception is returned when we have typecasting issue.